src/controllers/users.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const {
+  getAllUsers,
+  addUser,
+  getUser,
+  deleteUser,
+  updateUser,
+} = require('../services/usersServer');
+const { normalize } = require('../models/user');
+
+async function getAllUsersAction(req, res) {
+  const users = await getAllUsers();
+
+  res.send(users.map(user => normalize(user)));
+};
+
+async function addUserAction(req, res) {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const user = await addUser(name);
```

Comment:
here can happen errors. Let's handle all cases for all other controllers to avoid uncatched errors

Reply:
done